common/client/src/main/java/com/bazaarvoice/emodb/client/uri/EmoUriBuilder.java

```diff
@@ -516,6 +522,42 @@ public UriBuilder fragment(String fragment) {
         return this;
     }
 
+    @Override
+    public UriBuilder resolveTemplate(String name, Object value) {
+        // EMODB-MODIFICATION:  Templates are not supported, so buildFromMap is not supported
```

Comment:
nitpick, comment and exception don't correspond to method name

Reply:
updated for all URI template related comment and exception